mobileapp/redux/fetching.tsx

```diff
@@ -77,7 +77,7 @@ export async function fetchDogs(
       `http://${config.backend.ip}:${config.backend.port}/lostdogs?${filtersString}`,
       {
         headers: {
-          token: cookies,
+          token: Authorization,
```

Comment:
Like here, name cookies was perfectly ok, it is this 'token' that should be changed

Reply:
Yeah, I just wanted to make it consistent everywhere, so that we don't mix it up with something else. Since in the mobile cookie = token = Authorization